foundry/foundry.py

```diff
@@ -618,12 +618,25 @@ def _load_data(self, file=None, source_id=None, globus=True):
                 self.dataset.dataframe = pd.read_json(
                     os.path.join(path, file)
                 )
-            except:
-                # Try to read individual lines instead
-                self.dataset.dataframe = pd.read_json(
-                    os.path.join(path, file), lines=True
-                )
-
+            except Exception:
+                pass
```

Comment:
you shouldn't need to pass on any of these exceptions, if they are raised you'll want them to bubble up. instead do the try/except chaining outlined in https://docs.python.org/3/tutorial/errors.html#handling-exceptions